go-controller/pkg/node/default_node_network_controller.go

```diff
@@ -1329,27 +1331,68 @@ func DummyNextHopIPs() []net.IP {
 	return nextHops
 }
 
-// setGlobalForwardingRule either enables or disables forwarding for
-// all IPv4 and IPv6 interfaces by setting sysctl parameters based on
-// config.Gateway.DisableForwarding
-func setGlobalForwardingRule() error {
-	setForwardingSysctl := func(value string) error {
-		stdout, stderr, err := util.RunSysctl("-w", fmt.Sprintf("net.ipv4.ip_forward=%s", value), fmt.Sprintf("net.ipv6.conf.all.forwarding=%s", value))
-		if err != nil || stdout != fmt.Sprintf("%s\n%s", fmt.Sprintf("net.ipv4.ip_forward = %s", value), fmt.Sprintf("net.ipv6.conf.all.forwarding = %s", value)) {
-			return fmt.Errorf("could not set the correct forwarding value for all ipv4 and ipv6 interfaces: stdout: %v, stderr: %v, err: %v",
+// configureGlobalForwarding configures the global forwarding settings.
+// For IPv4 it either enables or disables forwarding for
+// all interfaces based on config.Gateway.DisableForwarding.
+// For IPv6 it always enables the global forwarding and adds/removes the `-A FORWARD -j DROP` rule based on
+// config.Gateway.DisableForwarding.
+func configureGlobalForwarding() error {
+	setIPv4Forwarding := func(value string) error {
+		stdout, stderr, err := util.RunSysctl("-w", fmt.Sprintf("net.ipv4.ip_forward=%s", value))
+		if err != nil || stdout != fmt.Sprintf("net.ipv4.ip_forward = %s", value) {
+			return fmt.Errorf("could not set the correct forwarding value for all ipv4 interfaces: stdout: %v, stderr: %v, err: %v",
```

Comment:
flavio's favorite nit: 😅
please consider using %w for err, so it can be used in wrap